content/en/docs/concepts/security/multi-tenancy.md

```diff
@@ -422,8 +422,7 @@ Priority and Fairness, is also recommended.
 
 Namespace isolation is well-supported by Kubernetes, has a negligible resource cost, and provides
 mechanisms to allow tenants to interact appropriately, such as by allowing service-to-service
-communication. However, it can be difficult to configure, and doesn't apply to Kubernetes
-resources that can't be namespaced, such as Custom Resource Definitions, Storage Classes, and Webhooks.
+communication. However, it can be difficult to configure, and it doesn't apply to resources that are cluster-scoped and not namespaced, such as Custom Resource Definitions (CRDs) themselves (though the custom resources they define can be namespaced), Storage Classes, and Webhooks.
```

Comment:
When we edit existing text, it's good to follow our style guide (no need to fix the rest of the page, although those fixes are also welcome).
Try this:
```suggestion
communication. However, it can be difficult to configure, and it doesn't apply to Kubernetes resources
that aren't namespaced, such as APIServices, StorageClasses, or ClusterTrustBundles.
```

(there's no Webhook API, StorageClass doesn't have a space, and CRD is a poor choice of example as it can mislead)

Reply:
Thank you for the feedback, @sftim I've made the suggested changes to align with the Kubernetes style guide. The examples now use APIServices, StorageClasses, and ClusterTrustBundles instead of CRDs and Webhooks to avoid confusion.

Let me know if further adjustments are needed!